packages/devextreme-cli/src/templates-ts/react/application/src/contexts/auth.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect, createContext, useContext, useCallback } from 'react';
+import { getUser, signIn as sendSignInRequest } from '../api/auth';
+
+interface IUser {
+  email: string;
+  avatarUrl: string;
+}
+
+type IAuthContextType = {
+  user?: IUser;
+  signIn: (email: string, password: string) => Promise<{isOk: boolean, data?: IUser, message?: string}>;
+  signOut: () => void;
+  loading: boolean;
+}
+
+interface IAuthProviderProps {
```

Comment:
Don't add new type children: React.ReactNode; use existing

Reply:
Ok, I will fix it in the next pr